app/controllers/application_controller.rb

```diff
@@ -473,9 +473,8 @@ def build_ae_tree(type = :ae, name = :ae_tree)
       end
     end
 
-    tree = TreeBuilderAutomate.new(name, type, @sb)
-    @automate_tree = tree if name == :automate_tree
```

Comment:
@ZitaNemeckova https://github.com/ZitaNemeckova/manageiq/blob/34861b855dd1c64e05f952635da2d980b8ef163a/app/views/layouts/_ae_tree_select.html.haml#L23 expects  ``` @automate_tree ``` to be set, removing ``` @automate_tree ``` can cause other issues.